ToolBox/AccernDataSplitter/config.json

```diff
@@ -0,0 +1,11 @@
+﻿{
+    "environment": "desktop",
+
+	"environments": {
+		"desktop": {}
+	},
+
+	// Delete the configuration after loading
+	"accern-data-folder": "",
```

Comment:
I know I'm being nit-picky, but this is the input folder, correct? I had to trace the value to prove that to myself, perhaps we rename this guy to `accern-input-data-folder` or even `accern-input-data-directory` -- also, do we need a specific location for the cache directory? Since it's a cache can't we just throw it into `accern-input-data-directory`/`cache` or something similar -- less required configuration is always a good thing :) If we do this, the code would need to create it at start (it's OK to create a dir that already exists, no error would be thrown).

Also, the comment on line 8 seems out of place.